base/strings/utf8proc.jl

```diff
@@ -431,13 +470,37 @@ ispunct(c::Char) = (UTF8PROC_CATEGORY_PC <= category_code(c) <= UTF8PROC_CATEGOR
 Tests whether a character is any whitespace character. Includes ASCII characters '\\t',
 '\\n', '\\v', '\\f', '\\r', and ' ', Latin-1 character U+0085, and characters in Unicode
 category Zs.
+
+# Examples
+```jldoctest
+julia> isspace('\n')
```

Comment:
Perhaps something like

```
julia> isspace.(['\n', '\r', ' ', '\x20'])
4-element BitArray{1}:
 true
 true
 true
 true
```

saves some space. But it might also just complicate things :)

Reply:
`islower` and `isupper` had 3 each before I got to the file. We could always remove one of my examples to match, if 3 is desirable.